CMakeLists.txt

```
@@ -2,10 +2,9 @@ cmake_minimum_required(VERSION 3.11)
 project(mlpack C CXX)
 
 include(CMake/CheckHash.cmake)
-include(CMake/Autodownload.cmake)
 include(CMake/ConfigureCrossCompile.cmake)
 include(CMake/CheckAtomic.cmake)
-
+include(CMake/Findmlpack.cmake)
```

```suggestion
include(CMake/Findmlpack.cmake)

```
(Just re-adding some spacing.)

---

I will change the naming of the file as well